src/main/java/com/scottlogic/sorts/KeywordSearch.java

```diff
@@ -0,0 +1,54 @@
+package com.scottlogic.sorts;
+
+import com.scottlogic.UserPost;
+import com.scottlogic.filters.ContentFilter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KeywordSearch {
```

Comment:
This class is confusing me a bit. Is it doing a search or a sort? If it is sorting, would it be better as a private method in `TopicSort`?

Reply:
It was in the instructions to make a new class: _"Keyword - Given a keyword return the posts ordered by the number of times the keyword appears in the Post content"_